src/engraving/libmscore/stafftypechange.cpp

```diff
@@ -139,6 +149,13 @@ void StaffTypeChange::draw(mu::draw::Painter* painter) const
     if (score()->printing() || !score()->showUnprintable()) {
         return;
     }
+
+    if (score()->isPaletteScore()) {
+        painter->setFont(m_iconFont);
+        painter->drawText(bbox(), draw::AlignCenter, Char(static_cast<char16_t>(ui::IconCode::Code::STAFF_TYPE_CHANGE)));
+        return;
+    }
```

Comment:
Is it possible to somehow move this logic to the palette module so that we have less code associated with palettes in the engraving module?

Reply:
@igorkorsukov I went for a completely new approach which was way easier. Thank you!